src/onegov/org/templates/occurrence.pt

```diff
@@ -14,10 +14,15 @@
                     <div tal:condition="occurrence.event.location">${occurrence.event.location}</div>
                 </div>
 
-                <div class="occurrence-tags">
+                <div class="occurrence-tags" tal:condition="occurrence.tags and not winti">
```

Comment:
Same here, don't have `onegov.winterthur` specific code in `onegov.org`

Reply:
done